contrib/platform/src/com/sun/jna/platform/win32/COM/AccessibleUtil.java

```diff
@@ -0,0 +1,88 @@
+/* Copyright (c) 2021 Mo Beigi, All Rights Reserved
+ *
+ * The contents of this file is dual-licensed under 2
+ * alternative Open Source/Free licenses: LGPL 2.1 or later and
+ * Apache License 2.0. (starting with JNA version 4.0.0).
+ *
+ * You can freely decide which license you want to apply to
+ * the project.
+ *
+ * You may obtain a copy of the LGPL License at:
+ *
+ * http://www.gnu.org/licenses/licenses.html
+ *
+ * A copy is also included in the downloadable source code package
+ * containing JNA, in file "LGPL2.1".
+ *
+ * You may obtain a copy of the Apache License at:
+ *
+ * http://www.apache.org/licenses/
+ *
+ * A copy is also included in the downloadable source code package
+ * containing JNA, in file "AL2.0".
+ */
+package com.sun.jna.platform.win32.COM;
+
+import com.sun.jna.platform.win32.OleAuto;
+import com.sun.jna.platform.win32.Variant;
+import com.sun.jna.platform.win32.Variant.VARIANT;
+import com.sun.jna.platform.win32.WTypes.BSTRByReference;
+import com.sun.jna.platform.win32.WinDef.LONG;
+import com.sun.jna.platform.win32.WinNT.HRESULT;
+
+import static com.sun.jna.platform.win32.COM.COMUtils.S_FALSE;
+import static com.sun.jna.platform.win32.COM.COMUtils.S_OK;
+import static com.sun.jna.platform.win32.WinError.E_INVALIDARG;
+
+/**
+ * {@link Accessible} utility API
+ *
+ * @author Mo Beigi, me@mobeigi.org
+ */
+public class AccessibleUtil
+{
+    /** The Accessible **/
+    final private IAccessible accessible;
+
+    public AccessibleUtil(IAccessible accessible) {
+        this.accessible = accessible;
+    }
```

Comment:
Is this a good way for this util to work? You pass in an `Accessible` and then call the functions on it?
Alternatively, the util class could be static but you'd need to pass in an `accessible` object to each call?

Reply:
Looks like that's how `TypeInfoUtil` is mapped and seems sane.